scenarios/import_td_logs/README.md

```diff
@@ -0,0 +1,36 @@
+# Workflow: Import Treasure Data Logs from Data Landing Area
+This example shows how you can use workflow to ingest Treasure Data Logs From Data Landing Areas to your Treasure Data account. 
```

Comment:
Add a statement;
"This is Opt-in feature. Contact your Customer Success rep or Technical Support if you have an interest in this feature."

Reply:
Thanks! Will fix.